assets/embauche.html

```diff
@@ -50,7 +50,16 @@
 					<option>mois</option>
 					<option disabled>jour</option>
 					<option disabled>heure</option>
-				</select>.
+				</select>
+
+				sur la commune de
+				<input name="code_postal_entreprise" id="code_postal_entreprise" class="medium-width centered"
```

Comment:
Or we can also do nothing and wait for this to land in FF (bug has been recently [assigned](https://bugzilla.mozilla.org/show_activity.cgi?id=771493) and the current UX isn't so bad).